Dockerfile

```diff
@@ -86,10 +86,13 @@ FROM minimal as ipfs
 LABEL org.opencontainers.image.description="Fluence Node bundled with IPFS"
 LABEL dev.fluence.bundles.ipfs="${IPFS_VERSION}"
 
-ENV IPFS_PATH=/config/ipfs IPFS_LOG_DIR=/log/ipfs IPFS_LOGGING_FMT=nocolor
+ENV IPFS_PATH=/config/ipfs
```

Comment:
why one var is on single line, and other two are on the same?

Reply:
Grouping of related variables (logging config) on one line